frontend/.env.example

```
@@ -2,6 +2,8 @@
 APP_ENV=
 APP_URL=
 NEXT_PUBLIC_SITE_URL=
+APP_ORIGIN=https://example.test
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_ORIGIN key should go before the APP_URL key

(UnorderedKey)